woocommerce/woocommerce
# Fix Conflict with [product_page] and Block Editor

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes #42785.

### How to test the changes in this Pull Request:

1. Make sure the Block Editor is being used.
2. Add the [product_page] shortcode to a page.
3. Refresh the Block Editor page.
4. The Block Editor should load the page that is being edited, not the product from the shortcode.

### Changelog entry


#### Significance


#### Type


#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Fix conflict between [product_page] and the Block Editor.

</details>
